Sources/ShopifyCheckoutKit/Configuration.swift

```diff
@@ -37,14 +37,13 @@ public struct Configuration {
 
 	public var confetti = Configuration.Confetti()
 
-	public var preloading = Configuration.Preloading()
-
 	public var spinnerColor: UIColor = UIColor(red: 0.09, green: 0.45, blue: 0.69, alpha: 1.00)
 
 	public var backgroundColor: UIColor = .systemBackground
 
 	public var logger: Logger = NoOpLogger()
 
+	public var preloading = Configuration.Preloading()
```

Comment:
Not sure we need this anymore. Think I would just expose the `clearCache` method at the same level as `preload()`, `present()` etc

Reply:
It needs to be public so it can be accessed in the sample app like so:
```
		ShopifyCheckoutKit.configuration.preloading.invalidateCache()
```